tuneme/templates/core/tags/section_listing_homepage.html

```diff
@@ -12,7 +12,23 @@
     </div>
     <h6><a href="{% pageurl section %}">{{section.title}}</a></h6>
   </div>
-</div>
+    {% for sub_section in section.sections %}
+    <div class="block {{section.get_effective_extra_style_hints}}">
+      {% for article in sub_section.articles %}
+        {% if article.featured_in_latest %}
+          {% if article.image %}
+            <a href="{% pageurl article %}">{% image article.image width-60 %}</a>
+          {% endif %}
+          <h3>{{article.title}}</h3>
+          <p>{{article.subtitle}}</p>
+          {% get_comment_count for article as comment_count %}
+          <div class="comments-count"><a href="">{{ comment_count }} comment{{ comment_count|pluralize }}<span></span></a></div>
+          <a href="{% pageurl article %}" class="button">Read on</a>
```

Comment:
Please put "Read on" and "View Similar" in `{% trans ... %}` blocks so we can translate them at a later stage.